src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -798,26 +755,27 @@ public void LoadGeneric6(InputType inputType, ReaderType readerType)
         [Theory]
         public void LoadGeneric7(InputType inputType, ReaderType readerType, TransformType transformType, DocType docType)
         {
-            AppContext.SetSwitch("Switch.System.Xml.AllowDefaultResolver", true);
-
-            string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
+            using (new AllowDefaultResolverContext())
+            {
+                string expected = @"<?xml version=""1.0"" encoding=""utf-8""?><result><fruit>Apple</fruit><fruit>orange</fruit></result>";
 
-            FileStream s2;
+                FileStream s2;
 
-            // check immediately after load and after transform
-            if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
-            {
-                s2 = new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read);
-                s2.Dispose();
-                if (Transform("fruits.xml", transformType, docType) == 1)
+                // check immediately after load and after transform
+                if (LoadXSL("XmlResolver_Main.xsl", inputType, readerType) == 1)
                 {
-                    VerifyResult(expected);
                     s2 = new FileStream(FullFilePath("XmlResolver_Main.xsl"), FileMode.Open, FileAccess.Read, FileShare.Read);
```

Comment:
For this and some other tests I wrapped their original code under an using statement to ensure that their change of the global context did not leak outside them - they were breaking some of the tests that I re-enabled. However, the diff here seems like a changed a bunch of their code... see ignoring white space to have a look at them https://github.com/dotnet/corefx/pull/22414/files?w=1